experiment/ndt7/ndt7.go

```diff
@@ -89,13 +89,15 @@ func measure(
 	callbacks.OnProgress(0, fmt.Sprintf("server: %s", client.FQDN))
 	for ev := range ch {
 		testkeys.Download = append(testkeys.Download, ev)
-		percentage := ev.Elapsed / maxRuntime / 2.0
-		message := fmt.Sprintf(
-			"max-bandwidth (download) %s (RTT min/smoothed/var %.1f/%.1f/%.1f ms)",
-			humanize.SI(float64(ev.BBRInfo.MaxBandwidth), "bit/s"),
-			ev.BBRInfo.MinRTT, ev.TCPInfo.SmoothedRTT, ev.TCPInfo.RTTVar,
-		)
-		callbacks.OnProgress(percentage, message)
+		if ev.AppInfo != nil && ev.Origin == "client" {
+			elapsed := float64(ev.AppInfo.ElapsedTime)/1e06 // to seconds
+			percentage := elapsed/maxRuntime/2.0
```

Comment:
It's not immediately clear why this percentage is `/2.0`, perhaps a comment would help?